code-challenges.test.js

```diff
@@ -1,59 +1,156 @@
+        // To do this we must first select the character at the 0th index and capitalize just that with .toUpperCase(). Next, we want to slice the same value starting with the 1st index, and use the .toLowerCase() method. We join these two together and get an array of words that have their first character capitalized and the rest of the characters in lowercase.
+const capitalizer = (array) => {
+    return array.map(value => {
+        return value.charAt(0).toUpperCase() + value.slice(1).toLowerCase()
+    })
+}
+// $ yarn jest (results)
+    // Test Suites: 1 passed, 1 total
+    // Tests:       2 passed, 2 total
+        // 2 passes for the 2 problems so far
+    // Pass!
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
-// b) Create the function that makes the test pass.
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
-
 // a) Create a test with expect statements for each of the variables provided.
+// Pseudocode
+    // Begin with describe() keyword. Takes in as its parameter the function name (vowelFinder) as a string, followed with anonymous function syntax.
+    // Write it() statement that takes in as its parameter a descriptive string of what the function should do. Follow with anonymous function syntax.
+    // Write an expect statement per set of provided variables. Argument is the function (vowelFinder) that takes as its parameters the respective variables (vowelTester1/2/3). Append with .toEqual() method that takes as its argument the expected outcome per set of variables (vowelOutput1/2/3).
+    // Describe statement completed.
+    // Run $ yarn jest. Should see "ReferenceError: vowelFinder is not defined"
+describe("vowelFinder", () => {
+    it("takes in a string and logs the index of the first vowel", () => {
+        const vowelTester1 = "learn"
+        const vowelOutput1 = 1
+        expect(vowelFinder(vowelTester1)).toEqual(vowelOutput1)
+        const vowelTester2 = "academy"
+        const vowelOutput2 = 0
+        expect(vowelFinder(vowelTester2)).toEqual(vowelOutput2)
+        const vowelTester3 = "challenges"
+        const vowelOutput3 = 2
+        expect(vowelFinder(vowelTester3)).toEqual(vowelOutput3)
+    })
+})
+// $ yarn jest (results)
+    // ReferenceError: vowelFinder is not defined
+    // Test Suites: 1 failed, 1 total
+    // Tests:       1 failed, 2 passed, 3 total
+        // 2 passed from first two problems
+    // Good error!
+// b) Create the function that makes the test pass.
+// Can't seem to figure out what function to write to make the test pass. Here are my attempts.
```

Comment:
Don't forget to pseudocode to help break down the logic.  I do like the approach of using .search, however, search is often combined with something called regex for this particular use case.  Here is some great documentation to take a look at to explore it further: [Search method with Regex](https://www.techonthenet.com/js/string_search.php)